components/popover/src/Popover.js

```diff
@@ -43,7 +49,6 @@ export const Popover = ({ open: isOpen, children, ...props }) => {
 		}
 	};
 
-	// not sure if this is correct
 	useEffect(() => {
```

Comment:
This looks like a great opportunity for a custom hook like `useOutsideHook` we could export from core.

```jsx
const handler = () => {}
const ref = useOutsideClick(handler)

return <div ref={ref} />
```

Reply:
Good idea, ill add it in a separate pr.